low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -285,3 +309,142 @@ def get_fans_info(self):
 
             data.append(fan_dict)
         return data
+
+    def initialize_systemd(self):
+        """Initialize system bus and interface."""
+        self._bus = SystemBus()
+        self._systemd = self._bus.get_object(
+            SYSTEMD_BUS, "/org/freedesktop/systemd1")
+        self._manager = Interface(self._systemd, dbus_interface=MANAGER_IFACE)
+
+    @staticmethod
+    def initialize_cortx_build_info():
+        """Initialize Cortx build info."""
+        if os.path.isfile(CORTX_RELEASE_FACTORY_INFO):
+            cortx_build_info_file_path = "%s://%s" % (
+                CONFIG_SPEC_TYPE, CORTX_RELEASE_FACTORY_INFO)
+            Conf.load("cortx_build_info", cortx_build_info_file_path)
+
+    @staticmethod
+    def get_cortx_service_info():
+        """Get cortx service info in required format."""
+        service_info = []
+        # TODO Get list of services from HA
+        return service_info
+
+    def get_external_service_info(self):
+        """Get external service info in required format."""
+        service_info = []
+        external_services = set(Conf.get(
+            SSPL_CONF, f"{self.SERVICE_HANDLER}>{self.SERVICE_LIST}", []))
+        for service in external_services:
+            response = self.get_systemd_service_info(service)
+            if response is not None:
+                service_info.append(response)
+        return service_info
+
+    @staticmethod
+    def get_service_info_from_rpm(service, prop):
+        """Get service info from its corrosponding RPM."""
+        systemd_path_list = ["/usr/lib/systemd/system/",
+                             "/etc/systemd/system/"]
+        result = "NA"
+        for path in systemd_path_list:
+            unit_file_path = path + service
```

Comment:
Add an example here how a unit_file_path  would look like

Reply:
Added example for `unit_file_path`.